mlir/test/mlir-translate/emitc_classops.mlir

```diff
@@ -0,0 +1,36 @@
+// RUN: mlir-translate --mlir-to-cpp %s | FileCheck %s
+
+emitc.class @modelClass {
+    emitc.field @input_tensor : !emitc.array<1xf32> 
+    emitc.field @some_feature : !emitc.array<1xf32>  {emitc.opaque = ["some_feature"]}
```

Comment:
add a second test where the fields don't have the attribute - everything should work, except no reflection support.